cpp/ql/src/semmle/code/cpp/exprs/BuiltInOperations.qll

```diff
@@ -333,7 +569,18 @@ class BuiltInOperationIsTriviallyDestructible extends BuiltInOperation, @istrivi
 }
 
 /**
- * The `__is_trivially_assignable` type trait.
+ * The `__is_trivially_assignable` built-in function (used by some
+ * implementations of the `<type_traits>` header).
+ *
+ * Returns `true` if the assignment operator `C::operator =(const C& c)` is
+ * trivial (compiler-generated).  The  * generated code will have resembled a
```

Comment:
Misplaced `*`.  I'm not convinced the second sentence is important to explaining the QL anyway.

Reply:
Agreed.